Docs/component-presentation/input-presentation.css

```diff
@@ -0,0 +1,29 @@
+.input-display-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 2rem;
+}
+
+.input-display{
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    gap: 2rem;
+}
+
+.input-display-container pre code{
+    height: 15rem;
+    max-width: min-content;
+}
+
+.input-display-container pre code::-webkit-scrollbar {
+    display: none;
+}
+
+
+@media screen and (min-width:1000px) {
+    .input-display{
+        max-width: 720px;
```

Comment:
Is it intentional that you have used px inside media queries or 
should be in rem unit?

Reply:
This is by mistake. It should be in rems. Fixing it.